package.json

```
@@ -70,7 +70,7 @@
     "all": true
   },
   "peerDependencies": {
-    "semantic-release": ">=15.8.0 <18.0.0"
+    "semantic-release": ">=15.8.0 <19.0.0"
```

because of the `engines` change above, only v18 and above satisfy the needs of this package. 
```suggestion
    "semantic-release": ">=18.0.0"
```